### PR TITLE
Some typography considerations.

### PR DESCRIPTION
**Pull request by lucaboesch**
I've been skimming through the new dev docs and I like them a lot.
I noticed that the typography is used differently and it would be maybe a good thing to use the ’ or ' consistently.
But I'm maybe too picky.

<a href="https://gitpod.io/#https://github.com/moodle/devdocs/pull/88"></a>

### CODE REVIEW

**Review by sarjona — CHANGES_REQUESTED** [review]
Hi @lucaboesch!
Thanks a lot for looking at the new devdocs and raising this.
I think it's better to leave \' instead of replacing it with \` .
However, I've seen you' fixed a couple of things too (shoudl and it's -> its). So could you please remove the changes done with ` and leave only these in order to merge the patch? TIA! :-)

**Comment by lucaboesch** [comment]
Good morning @sarjona.
I see where you're getting at, so it would be of personal taste of the editor, with one person writing it is ` and with another it is '
We could also have the ` changed into ' …
What do you think?

**Comment by andrewnicols** [comment]
Personally I'm not keen on this change. We should use the single (`'`) for contractions (there's, it's, doesn't, etc.) because that is the key on the keyboard that one types. A standard English (both US and UK variants) has no apostrophe character (`’`) - to get one Mac OS one must use `Option + Shift + ]`.

The only options availble to us are:
- Backtick `
- Quote `"`
- Single quote `'`
- Apostrophe `’`

We _CANNOT_ use the backtick as that is used for code. We _SHOULD NOT_ use the double-quote, because that is jsut wrong.
Although the Apostrophe is probably more typographically correct, it's also extremely non-obvious for a vast majority of our devs and does not appear on standard English keyboards.

**Comment by lucaboesch** [comment]
Probably I've not gotten each and every \’ but I've switched those to '.

File: docs/guides/javascript/index.md [hunk 1]
```diff
@@ -325,9 +325,9 @@ A lot of this content is being migrated to use Mustache Templates which are the
 
 Where content is generated in PHP you will need to include your JavaScript at the same time.
 
-Although several older ways to include JavaScript from PHP, it`s strongly
-recommended that all new JavaScript only use the`js_call_amd` function on the
-`page_requirements_manager`.
+Although several older ways to include JavaScript from PHP, it's strongly
+recommended that all new JavaScript only use the \`js_call_amd\` function on the
```
Comment (sarjona) [inline docs/guides/javascript/index.md:329]:
```suggestion
recommended that all new JavaScript only use the `js_call_amd` function on the
```

File: docs/guides/javascript/index.md [hunk 2]
```diff
@@ -325,9 +325,9 @@ A lot of this content is being migrated to use Mustache Templates which are the
...
+\`page_requirements_manager\`.
```
Comment (sarjona) [inline docs/guides/javascript/index.md:330]:
```suggestion
`page_requirements_manager`.
```

**Review by sarjona — CHANGES_REQUESTED** [review]
Thanks Luca for making these apostrophes more consistent!! As you'll see, there are only a couple of places where Backtick shouldn't be escaped.

**Review by sarjona — CHANGES_REQUESTED** [review]
Hi Luca!
Thanks, the patch looks great! I have only comments about git: it would be great if you could squash your commits in 1 (I can't see the value of having separate commits for these changes). And it would also be amazing if you could reword it, adding "[docs]" as prefix (you can look at the history of this repository to see what I mean).
This is still a WIP and we are working on a page to explain how to contribute to this devdocs (with things like this to consider) :-)
Tons of kisses for you, Luca! :-*

**Comment by lucaboesch** [comment]
Done, @sarjona and thousands of kisses back!

File: docs/guides/javascript/index.md (hunk 1 above)
Comment (sarjona) [inline docs/guides/javascript/index.md:329]:
```suggestion
recommended that all new JavaScript only use the `js_call_amd` function on the
```

File: docs/guides/javascript/index.md (hunk 2 above)
Comment (sarjona) [inline docs/guides/javascript/index.md:330]:
```suggestion
`page_requirements_manager`.
```

**Review by sarjona — CHANGES_REQUESTED** [review]
Thanks Luca! We're almost there! There are still a couple of places where the \` shouldn't be escaped ;-)

(Remember to squash changes in a single commit, as you've done now!) And then, it will be ready to get integrated ;-)

**Comment by lucaboesch** [comment]
There you go!

**Review by sarjona — APPROVED** [review]

**Pull request merged**